### PR TITLE
DMX Bidfloor fix

### PR DESCRIPTION
**Pull request by stevealliance**
added bidfloor from original JSON request

### CODE REVIEW

**Review by guscarreon — DISMISSED** [review]
Addition looks great.

File: adapters/dmx/dmx.go [hunk 1]
```diff
@@ -221,21 +222,40 @@ func (adapter *DmxAdapter) MakeBids(request *openrtb.BidRequest, externalRequest
...
 func fetchParams(params dmxExt, inst openrtb.Imp, ins openrtb.Imp, imps []openrtb.Imp, banner *openrtb.Banner, video *openrtb.Video, intVal int8) []openrtb.Imp {
+
 	if params.Bidder.TagId != "" {
 		ins = openrtb.Imp{
-			ID:     inst.ID,
-			TagID:  params.Bidder.TagId,
-			Ext:    inst.Ext,
-			Secure: &intVal,
+			ID:                inst.ID,
+			TagID:             params.Bidder.TagId,
```
Comment (mansinahar) [inline adapters/dmx/dmx.go:229]: Wouldn't it be easier to just create a copy of `inst` as `ins` and then manipulate the fields that you need to manipulate? And you can do that outside of these two if conditions as everything else except the `tagID` value is the same in either case.

**Review by mansinahar — COMMENTED** [review]
Please add a JSON test to make sure these fields are propagated to your service as expected

File: adapters/dmx/dmx.go (hunk 1 above)
Comment (stevealliance) [reply adapters/dmx/dmx.go]: @mansinahar Added test json for imp other expected value for banner and video

**Review by guscarreon — APPROVED** [review]
LGTM

**Review by mansinahar — APPROVED** [review]

**Pull request merged**